readthedocs/rtd_tests/tests/test_config_integration.py

```diff
@@ -199,23 +211,582 @@ def test_conda(self, load_config):
             }
         })
         config = load_yaml_config(self.version)
-        self.assertEqual(config.use_conda, True)
-        self.assertTrue(config.conda_file[-len(to_find):] == to_find)
+        self.assertTrue(config.conda is not None)
+        self.assertTrue(config.conda.environment[-len(to_find):] == to_find)
 
         load_config.side_effect = create_load()
         config = load_yaml_config(self.version)
-        self.assertEqual(config.use_conda, False)
-        self.assertEqual(config.conda_file, None)
+        self.assertIsNone(config.conda)
 
     def test_requirements_file(self, load_config):
         requirements_file = '__init__.py'
         load_config.side_effect = create_load({
             'requirements_file': requirements_file
         })
         config = load_yaml_config(self.version)
-        self.assertEqual(config.requirements_file, requirements_file)
+        self.assertEqual(config.python.requirements, requirements_file)
 
         # Respects the requirements file from the project settings
         load_config.side_effect = create_load()
         config = load_yaml_config(self.version)
-        self.assertEqual(config.requirements_file, '__init__.py')
+        self.assertEqual(config.python.requirements, '__init__.py')
+
+
+@pytest.mark.skip
```

Comment:
Remember to un-skip this before merging.

Reply:
This PR should skip those, the PRs that implement the actual logic for v2 #4493 #4482 #4486 un-skip these tests